src/main/webapp/app/entities/event-attendee/event-attendee-create.tsx

```diff
@@ -11,37 +12,39 @@ import { IRootState } from 'app/shared/reducers';
 import { getEntity, updateEntity, createEntity, reset } from './event-attendee.reducer';
 import { IEventAttendee } from 'app/shared/model/event-attendee.model';
 // tslint:disable-next-line:no-unused-variable
-import { convertDateTimeFromServer, convertDateTimeToServer } from 'app/shared/util/date-utils';
-import { mapIdList } from 'app/shared/util/entity-utils';
+import { IEvent } from 'app/shared/model/event.model';
 
-export interface IEventAttendeeUpdateProps extends StateProps, DispatchProps, RouteComponentProps<{ id: string }> {}
+import './event-attendee-create.scss';
+
+export interface IEventAttendeeUpdateProps extends StateProps, DispatchProps, RouteComponentProps<{ id: string; eventId: string }> {}
 
 export interface IEventAttendeeUpdateState {
-  isNew: boolean;
+  event: IEvent;
 }
 
 export class EventAttendeeUpdate extends React.Component<IEventAttendeeUpdateProps, IEventAttendeeUpdateState> {
   constructor(props) {
     super(props);
     this.state = {
-      isNew: !this.props.match.params || !this.props.match.params.id
+      event: null
     };
   }
 
+  getEvent = async () => {
+    const event = await axios.get<IEvent>(`api/events/${this.props.match.params.eventId}`);
+    this.setState({ event: event.data });
+  };
```

Comment:
can have an enhancement to put this in reducer?

Reply:
Sure, created an enhancement issue in #103